gravitee-apim-console-webui/src/management/integrations/integrations.model.ts

```diff
@@ -16,12 +16,20 @@
 
 import { Pagination } from '../../entities/management-api-v2';
 
+export interface IntegrationNavigationItem {
+  routerLink: string;
+  displayName: string;
+  permissions: string[];
+  icon: string;
+}
+
 export interface IntegrationResponse {
   data: Integration[];
   pagination: Pagination;
 }
 
 export interface Integration {
```

Comment:
I see we keep all entities in one place (src/entities) so I guess I would move also integration entities there

Reply:
hmm, not all, see some .model.ts files in other places. I slightly prefer keep model related to module in the same directory. However I can move without complaining :-) , @jgiovaresco WDYT ? 
![image](https://github.com/gravitee-io/gravitee-api-management/assets/135582189/25fe8568-c14b-4efe-8d95-79eea76d711a)

Reply:
For the best should be to have 2 models 🙈 :
- the one in `entities` that matches with the backend contract
- another one in the module with an adaption layer to move from one to another in the service calling the backed

However, this should require a lot a change to apply that everywhere... 

Defining models in their own module seems a good alternative. Just to be sure, maybe we can ask other teams for their opinion (for the sake of consistency) but I'm good with that

Reply:
I asked, no reply so far. I would merge it as it is in the module, this is common Angular pattern.

Reply:
Thibaud, and Jourdi did not mind this approuch in one of my pull requests in Migration Team, so I gues this is OK.